spec/models/spotlight/blacklight_configuration_spec.rb

```diff
@@ -452,9 +452,9 @@
   describe '#custom_index_fields' do
     it 'converts exhibit-specific fields to Blacklight configurations' do
       allow(subject.exhibit).to receive_messages(custom_fields: [
-        stub_model(Spotlight::CustomField, field: 'abc', configuration: { a: 1 }, exhibit: subject.exhibit),
-        stub_model(Spotlight::CustomField, field: 'xyz', configuration: { x: 2 }, exhibit: subject.exhibit)
-      ])
+                                                   stub_model(Spotlight::CustomField, field: 'abc', configuration: { a: 1 }, exhibit: subject.exhibit),
+                                                   stub_model(Spotlight::CustomField, field: 'xyz', configuration: { x: 2 }, exhibit: subject.exhibit)
+                                                 ])
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.